### PR TITLE
Enable mc build tests on go1.9

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,8 @@ env:
 - ARCH=i686
```
Comment (balamurugana) [inline .travis.yml]: Is 32 bit required?

File: .travis.yml [hunk 2]
```diff
@@ -9,7 +9,8 @@ env:
...
 
 go:
-- 1.8
+- 1.8.x
```
Comment (balamurugana) [inline .travis.yml]: Do we need to backward compatible?

File: .travis.yml (hunk 1 above)
Comment (harshavardhana) [reply .travis.yml]: mc is not decided yet @balamurugana as a client perhaps its okay so we kept it.

Comment (harshavardhana) [reply .travis.yml]: Actually you are right we don't need.. removing.

File: .travis.yml (hunk 2 above)
Comment (harshavardhana) [reply .travis.yml]: We do not have to be, but mc has been allowed to be built since pretty much 1.7

**Pull request merged**